.github/workflows/build-push.yml

```diff
@@ -2,42 +2,84 @@ name: BuildPushDeployImage
 
 on:
   push:
-
-env:
-  IMAGE_VERSION: '0.7'
-  IMAGE_REGISTRY: quay.io/sustainable_computing_io
-  IMAGE_NAME: kepler_model_server
+    branches: [ main ]
 
 jobs:
-  build:
+
+  changes:
     runs-on: ubuntu-latest
+    outputs:
+      src: ${{ steps.changes.outputs.src }}
+      infra: ${{ steps.changes.outputs.infra }}
```

Comment:
I don't see infra map below. 
I think we need to remove it.

Reply:
updated.